content-src/activity-stream.jsx

```diff
@@ -11,15 +11,11 @@ const store = initStore(reducers);
 
 new DetectUserSessionStart(store).sendEventOrAddListener();
 
-// If we are starting in a prerendered state, we must wait until the first render
-// to request state rehydration (see Base.jsx). If we are NOT in a prerendered state,
-// we can request it immedately.
 store.dispatch(ac.AlsoToMain({type: at.NEW_TAB_STATE_REQUEST}));
```

Comment:
This used to be dispatched after first render of base. Should we restore that.. with hooks? ;) https://github.com/mozilla/activity-stream/commit/5e613f4060564cb206cd28e5e2ecbeda7b1950c0#diff-1bc118e550611e1215c06a438572d1a4L26